miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/RealMiniApp.kt

```diff
@@ -23,6 +23,17 @@ internal class RealMiniApp(
     override suspend fun create(
         info: MiniAppInfo,
         miniAppMessageBridge: MiniAppMessageBridge
+    ): MiniAppDisplay = executingCreate(info, miniAppMessageBridge)
+
+    @Suppress("TooGenericExceptionThrown")
+    override suspend fun create(info: MiniAppInfo): MiniAppDisplay =
+        executingCreate(info, object : MiniAppMessageBridge() {
+            override fun getUniqueId(): String = throw Exception()
```

Comment:
Probably this Exception should have some error message. Something like `MiniAppMessageBridge has not been implemented`?